addon/components/infinity-loader.js

```diff
@@ -168,6 +168,7 @@ const InfinityLoaderComponent = Component.extend(InViewportMixin, {
       .then((infinityModel) => {
         infinityModel.on('infinityModelLoaded', this, this._loadStatusDidChange);
         set(infinityModel, '_scrollable', get(this, 'scrollable'));
+        set(this, 'isDoneLoading', false);
```

Comment:
This property probably needs to be watched inside the `infinity-loader` helper #395

Reply:
the helper?

Reply:
Sorry I meant component. But wait, this _is_ the component...

I mean that when the model changes (and the number of results changes), the InfinityModel no longer reached infinity, so hideOnInfinity should be undone when it was reached.

I guess this would entail setting `set(this, 'isDoneLoading', false)` and `set(this, 'isVisible', true)`, or - if possible - rerun `_initialInfinityModelSetup()`.

I'll explain what I mean in #395